### PR TITLE
Adds the pondering orb to the wizard den

### PR DESCRIPTION
**Pull request by datlo**
## What Does This PR Do

Adds a stationary pondering orb to the wizard den, letting wizards scout the station and ponder their plans with observers. Works similarly to the scrying orb, but it can't be taken with you.

This also removes the teleport scroll landmark as it's not used anywhere.
## Why It's Good For The Game

I think helping wizards prepare will help less experienced wizards last longer on the field. Although this will also buff robust wizard players, they already know the map like the back of their hand and can adapt to situations better, so I don't think this will cause significant balance issues.
ad3

## Images of changes

image

This is a placeholder sprite, to be replaced with a proper structure.

## Todo
- Add sprite

## Testing

Spawned in as a wizard, tested the pondering orb and the scrying orb. Spawned in as a cultist, tested spirit realm rune. All working as intended.

## Changelog
:cl:
add: Adds the pondering orb to the wizard den, letting wizards ponder their plans before teleporting stationside.
/:cl:

### CODE REVIEW

File: code/game/objects/structures/pondering_orb.dm [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+			user.grab_ghost()
+			user.visible_message("<span class='notice'>[user]'s focus is forced away from [src].</span>",
+								"<span class='userdanger'>Your vision is ripped away from [src].</span>")
+			break
+		sleep(5)
+	in_use = FALSE
+	if(QDELETED(user))
+		return
+	user.remove_atom_colour(ADMIN_COLOUR_PRIORITY, COLOR_BLUE)
+	REMOVE_TRAIT(user, SCRYING, SCRYING_ORB)
```
Comment (Contrabang) [inline code/game/objects/structures/pondering_orb.dm:33]: This is the same code as the one from the other orb, can you make a general proc for this or something?

**Review by Contrabang — CHANGES_REQUESTED** [review]

File: code/game/objects/structures/pondering_orb.dm (hunk 1 above)
Comment (datlo) [reply code/game/objects/structures/pondering_orb.dm:33]: yeah thats a good idea i'll do that

**Comment by datlo** [comment]
Can anyone tell me why I'm getting thousand line changes when I'm only changing a single tile on wiz den?
I ran run before commiting bat script

**Comment by Henri215** [comment]
image

File: code/game/objects/structures/pondering_orb.dm [hunk 2]
```diff
@@ -0,0 +1,28 @@
+/obj/structure/pondering_orb
+	name = "pondering orb"
+	icon = 'icons/obj/wizard.dmi'
+	desc = "Ponder your plans for the station using this farsight artefact."
```
Comment (Henri215) [inline code/game/objects/structures/pondering_orb.dm:4]:
```suggestion
	desc = "Ponder your plans for the station using this farsight artefact."
	icon = 'icons/obj/wizard.dmi'
```

File: code/game/objects/structures/pondering_orb.dm [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+	icon_state = "scrying_orb" //TODO : Placeholder sprite
+	anchored = TRUE
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	var/mob/dead/observer/ghost // owners ghost when active
+
+/obj/structure/pondering_orb/attack_hand(mob/user)
+	ADD_TRAIT(user, SCRYING, SCRYING_ORB)
+	user.add_atom_colour(COLOR_BLUE, ADMIN_COLOUR_PRIORITY) // stolen spirit rune code
+	user.visible_message("<span class='notice'>[user] stares into [src], [user.p_their()] eyes glazing over.</span>",
+					"<span class='danger'> You stare into [src], you can see the entire universe!</span>")
```
Comment (Henri215) [inline code/game/objects/structures/pondering_orb.dm:14]:
```suggestion
					"<span class='danger'>You stare into [src], you can see the entire universe!</span>")
```

**Comment by Sirryan2002** [comment]
convert this to a WIP PR please

**Comment by datlo** [comment]
This is ready for review and could be merged as-is although it's using the scrying orb for sprite.

File: code/game/objects/structures/pondering_orb.dm [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+	resistance_flags = INDESTRUCTIBLE | LAVA_PROOF | FIRE_PROOF | ACID_PROOF
+	var/mob/dead/observer/ghost // owners ghost when active
+
+/obj/structure/pondering_orb/attack_hand(mob/user)
+	ADD_TRAIT(user, SCRYING, SCRYING_ORB)
+	user.visible_message("<span class='notice'>[user] stares into [src], [user.p_their()] eyes glazing over.</span>",
+					"<span class='danger'>You stare into [src], you can see the entire universe!</span>")
+	ghost = user.ghostize(TRUE, COLOR_BLUE, "Magic Spirit of [user.name]")
+	while(!QDELETED(user))
+		if(user.key || QDELETED(src))
```
Comment (Contrabang) [inline code/game/objects/structures/pondering_orb.dm:16]: You should make sure that theyre adjacent too it also

**Review by S34NW — APPROVED** [review]

**Pull request merged**